src/main/scala/Bintray.scala

```diff
@@ -57,36 +54,46 @@ object Bintray {
 
   private[bintray] def ensuredCredentials(
     credsFile: File, prompt: Boolean = true): Option[BintrayCredentials] =
-    BintrayCredentials.read(credsFile).fold(sys.error(_), _ match {
-      case None =>
-        if (prompt) {
-          println("bintray-sbt requires your bintray credentials.")
-          saveBintrayCredentials(credsFile)(requestCredentials())
-          ensuredCredentials(credsFile, prompt)
-        } else {
-          println(s"Missing bintray credentials $credsFile. Some bintray features depend on this.")
-          None
+      propsCredentials("bintray")
+        .orElse(envCredentials("bintray"))
+        .orElse(BintrayCredentials.read(credsFile)) match {
+          case None =>
+            if (prompt) {
+              println("bintray-sbt requires your bintray credentials.")
```

Comment:
There are some more `println` usages than here. I wanted to have this PR as small as possible, so we can merge it sooner rather than later. Getting rid of all `println` as other cleanups could be done as a separate PR.